horreum-integration/src/test/java/io/hyperfoil/tools/HorreumClientTest.java

```diff
@@ -13,9 +13,11 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.extension.ExtendWith;
 
+@ExtendWith({HorreumTestExtension.class})
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
-public class HorreumClientTest extends HorreumTestBase {
+public class HorreumClientTest extends HorreumTestClientExtension {
```

Comment:
Should it extend `HorreumTestClientExtension`, or just use `@ExtendWith(HorreumTestClientExtension.class)`? I am a bit confused here...

Reply:
In the desperation to get something working I munged together composition and inheritance. I'll separate it out for clarity. Thanks for spotting this.